src/scale-manager/README.md

```diff
@@ -1,23 +1,115 @@
 # Scale Manager
-A utility class that listens for resize events.
-
-There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
+A utility class that listens for resize events and calculates the width and height your game should be to fit within the screen without stretching or squishing the game. 
 
 ```javascript
-var scaleManager = new ScaleManager(function(resizeData) {
-  console.log('This is called on window resize');
+let scaleManager = new ScaleManager({
+  width: 1320,
+  height: 780,
+  callback: function(resizeData) {
+    console.log('This is called on window resize');
 
-  console.log('Window width is', resizeData.width);
-  console.log('Window height is', resizeData.height);
-  console.log('Window aspect ratio is', resizeData.ratio);
+    console.log('game width is', resizeData.width);
+    console.log('game height is', resizeData.height);
+    console.log('game scale is', resizeData.scale);
+  }
 });
 ```
 
-The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the
-existing listener.
+The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the existing listener. 
 
 ```javascript
 scaleManager.enable(function(resizeData) {
   console.log('The old listener is replaced with this one!', resizeData);
 });
 ```
+
+**Note:** There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
+
+### Callback Example
+
+The scale manager callback parameter contains `width`, `height`, and `scale`. these values can be used to resize your game. Depending on the engine you choose you'll have to use them differently. `scale` contains `x` and `y`, these are the independent scaling values for the x and y axis.
+
+```javascript
+scaleManager.enable(({width, height, scale}) => {
+  // -- PIXI -- //
+  const view = this.pixi.view;
+  const stage = this.pixi.stage;
+  const renderer = this.pixi.renderer;
+
+  stage.position.set(renderer.width / 2, renderer.height / 2);
+  stage.scale.set(scale.x, scale.y);
+
+  stage.pivot.x = renderer.width / 2;
+  stage.pivot.y = renderer.height / 2;
+
+  view.style.width = width + 'px';
+  view.style.height = height + 'px';
+
+  view.style.position = 'absolute'
+  view.style.left = '0px';
+  view.style.top = '0px'; 
+});
+```
+
+**Note**: A Phaser3 example can be found here: https://github.com/SpringRoll/Springroll-Seed/tree/templates/pixi
```

Comment:
This link appears to be incorrect. Moreover, you should probably format this to be use markdown:

```markdown
can be found [here](https://github.com/SpringRoll)
```